owlbot.py

```diff
@@ -36,15 +36,6 @@
         "scripts/fixup_documentai_v*",  # this library was always generated with the microgenerator
     ]
 
-    s.replace(library / "google/cloud/documentai_v1beta2/**/base.py",
-        """scopes_kwargs = \{"scopes": scopes, "default_scopes": cls\.AUTH_SCOPES\}""",
-        """# Documentai uses a regional host (us-documentai.googleapis.com) as the default
-            # so self-signed JWT cannot be used.
-            # Intentionally pass default scopes as user scopes so the auth library
-            # does not use the self-signed JWT flow.
-            # https://github.com/googleapis/python-documentai/issues/174
-            scopes_kwargs = {"scopes": scopes or cls.AUTH_SCOPES, "default_scopes": cls.AUTH_SCOPES}""")
-
     s.replace(library / "tests/**/documentai_v1beta2/*.py",
```

Comment:
I believe you'll want to also delete the modifications to the unit tests. After that, please re-run owlbot using the commands @parthea shared.

Reply:
Done.